api/src/main/java/io/minio/MinioClient.java

```diff
@@ -86,13 +84,9 @@
 import java.nio.file.Paths;
 import java.nio.file.StandardCopyOption;
 import java.nio.file.StandardOpenOption;
-import java.security.InvalidAlgorithmParameterException;
 import java.security.InvalidKeyException;
-import java.security.Key;
-import java.security.KeyPair;
 import java.security.KeyManagementException;
 import java.security.NoSuchAlgorithmException;
```

Comment:
Remove the above two imports.

Reply:
`NoSuchAlgorithmException` is used at other places as well, like https://github.com/minio/minio-java/blob/master/api/src/main/java/io/minio/Signer.java#L369